types/formats.ts

```diff
@@ -0,0 +1,130 @@
+/** Time value in seconds */
+export type Seconds = number;
+
+/** Pixels value */
+export type Pixels = number;
+
+/** Proportion percent value between 0 and 1 */
+export type ProportionPercent = number;
+
+/** Volume level between 0 and 1 */
+export type VolumeLevel = number;
+
+/** Playback rate between 0.5 and 2 */
+export type PlaybackRate = number;
+
+/** Hex color code without the hash (e.g. '00adef') */
+export type HexColor = string;
```

Comment:
Same as above, for both 0.5-2 and valid hex values.

Reply:
I would also check if they're numbers or "0/1" strings